src/app/accounts/login.component.html

```diff
@@ -1,7 +1,9 @@
 <form #form (ngSubmit)="submit()" [appApiAction]="formPromise" class="container" ngNativeValidate>
     <div class="row justify-content-md-center mt-5">
         <div class="col-5">
-            <img src="../../images/logo-dark@2x.png" class="logo mb-2" alt="Bitwarden">
```

Comment:
I thought that we used `img` here because it's semantic, it's not just a background image, it's the title/header image for the page and it should get picked up by screen readers and other assistive technologies.

We use CSS `background-image` for `index.html` but I consider that a hack that I didn't really want to replicate elsewhere (even though that would at least be consistently hacky 😆 )

I like your idea of a pipe though, I'll investigate that.

Reply:
Update: I realised that this is a solved problem in the desktop client, so I used the same solution here. We can use scss to set the image, using the `content` attribute. That keeps it really simple while also being able to keep the semantic `img` tag with its `alt` atribute for accessibility.